src/hash.cr

```diff
@@ -51,6 +51,13 @@ class Hash(K, V)
     new nil, comparator
   end
 
+  # Set the value given by *value* with the key given by *key*.
```

Comment:
That sounds a bit backwards to me tbh, "Set the value of *key* to the given *value*" perhaps?